Documentation/gettingstarted/cli-status.rst

```diff
@@ -1,17 +1,8 @@
-Validate the Installation
-=========================
-
-.. include:: install-cli.rst
-
 To validate that Cilium has been properly installed, you can run
 
 .. code-block:: shell-session
 
-   cilium status --wait
-
-The output should be similar to the following one:
-
-::
+   $ cilium status --wait
```

Comment:
No blank line necessary between command and output.

Reply:
Fixed in latest push.